setup.py

```diff
@@ -24,9 +24,12 @@
 with io.open("Readme.md", "r", encoding="utf8") as fh:
     long_description = fh.read().strip()
 
+sources = glob("src/*.cpp") + glob("src/syzygy/*.cpp")
+sources.remove('src/ffishjs.cpp')
```

Comment:
I think this `remove` does not work on Windows (which is why appveyor is failing) since although the glob has to use Unix-style paths, the returned lists of paths are OS specific. So using `os.path.join` might work, or if you want to be on the save side you can use a glob again to definitely get the same path.